pkg/route/api/allocator/allocator.go

```diff
@@ -0,0 +1,52 @@
+package allocator
+
+import (
+	"fmt"
+	"regexp"
+
+	"code.google.com/p/go-uuid/uuid"
+	routeapi "github.com/openshift/origin/pkg/route/api"
+)
+
+// This should be something we get from config but we would still need a
+// default name if nothing's passed. The first pass uses the default name.
+// Would be better if we could use "v3.openshift.app", someone bought that!
+const DEFAULT_DNS_SUFFIX = "v3.openshift.com"
```

Comment:
Constants should be in "CamelCase" if public and "camelCase" if not (since this should be removed anyway, just fyi)